test/e2e/servers/graphql-main-server/galoy-pay-testing.spec.ts

```diff
@@ -231,6 +224,12 @@ describe("galoy-pay", () => {
       })
       expect(makePayment.data.lnInvoicePaymentSend.status).toEqual("SUCCESS")
 
+      // Subscribe to the invoice
```

Comment:
Why did this have to be moved to after the invoice is paid?

Reply:
because of a race condition.. I had added sleep but in this way is cleaner. I had random errors with github actions

Reply:
adding status query before subscription paid validation gives enough time to fix the race condition with `notificationsService.lightningTxReceived` (no await)
https://github.com/GaloyMoney/galoy/pull/2534/commits/c76a2fbc0ffc579a303efcd86bc9d0213323c5d0